core/src/main/java/edu/stanford/futuredata/macrobase/pipeline/BasicBatchPipeline.java

```diff
@@ -76,6 +78,15 @@ public BatchSummarizer getSummarizer(String outlierColumnName) throws MacrobaseE
                 summarizer.setMinRiskRatio(minRiskRatio);
                 return summarizer;
             }
+            case "fpgrowth": {
+                FPGrowthSummarizer summarizer = new FPGrowthSummarizer();
+                summarizer.setOutlierColumn(outlierColumnName);
+                summarizer.setAttributes(attributes);
+                summarizer.setMinSupport(minSupport);
+                summarizer.setMinRiskRatio(minRiskRatio);
+                summarizer.setUseAttributeCombinations(true);
+                return summarizer;
+            }
```

Comment:
I was actually about to submit another PR to set a maximum order LOL. @edgan8 do you wanna incorporate it in this PR, or should I push mine?